design-documents/graph-ql/coverage/Cart.graphqls

```diff
@@ -0,0 +1,59 @@
+type Query {
+    cart(input: CartQueryInput): CartQueryOutput
+}
+
+input CartQueryInput {
+    cart_id: String!
+}
+
+type CartQueryOutput {
+    cart: Cart
+}
+
+type Cart {
+    id: String!
+
+    line_items_count: Int!
+    items_quantity: Float!
+
+    selected_payment_method: CheckoutPaymentMethod
+    available_payment_methods: [CheckoutPaymentMethod]!
+
+    customer: CheckoutCustomer
+    customer_notes: String
+
+    gift_cards_amount_used: Money
+    applied_gift_cards: [CartGiftCard]
+
+    is_multishipping: Boolean!
+    is_virtual: Boolean!
+}
+
+type CheckoutCustomer {
+    is_guest: Boolean!
+    email: String!
+    prefix: String
+    first_name: String!
+    last_name: String!
+    middle_name: String
+    suffix: String
+    gender: GenderEnum
+    date_of_birth: String
```

Comment:
Are there any plans for a scalar `Date` type in the core?

Reply:
We do not have plans for that, but I know that this topic was discussed in scope of Webonyx project: https://github.com/webonyx/graphql-php/issues/228